internal/server/http/server_test.go

```diff
@@ -67,7 +70,7 @@ func TestBuildProxyMode(t *testing.T) {
 		imposterFs, err := NewImposterFS("test/testdata/imposters")
 		require.NoError(t, err)
 
-		server := NewServer(router, httpServer, proxyServer, false, imposterFs)
+		server := NewServer(router, httpServer, proxyServer, false, imposterFs, nil, 0, "")
```

Comment:
Smell that points to https://github.com/friendsofgo/killgrave/pull/170/files#r1686772983

Reply:
Now handled with a serverconfig